src/megatron/energon/task_encoder/base.py

```diff
@@ -207,6 +256,17 @@ def _batch(
         else:
             raise ValueError("Unrecognized result type.")
 
+    def slice_batch(self, samples: List[T_sample]) -> List[List[T_sample]]:
```

Comment:
When changing the "slice_batcher", this now operates on the "slice_buffer", thus rename this to "slice"

Reply:
TODO: Add slice buffer size, must be set somehow. Default to `0` to disable?